src/lib/components/table/Table.tsx

```diff
@@ -1,17 +1,22 @@
-import { TableCell, TableCellProps } from './components/TableCell';
+import { TableCell, TableCellProps } from './components/TableCell/TableCell';
 import { BottomOrRegularModal } from '../modal';
 import { ReactNode, useEffect, useRef, useState } from 'react';
-import { ChevronDownIcon, ChevronLeftIcon, ChevronRightIcon, ChevronUpIcon } from '../icon';
+import {
+  ChevronDownIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+  ChevronUpIcon,
+} from '../icon';
 import { Card } from '../cards/card';
 
-import styles from './style.module.scss';
+import styles from './Table.module.scss';
 import { useMediaQuery } from '../../hooks/useMediaQuery';
-import { TableSection } from './components/TableSection';
+import { TableSection } from './components/TableSection/TableSection';
 import classNames from 'classnames';
-import { useTableNavigation } from './utils/useTableNavigation';
-import { TableControls } from './components/TableControls';
-import { TableContent } from './components/TableContent';
-import { useScrollSync } from './utils/useScrollSync';
+import { useTableNavigation } from './utils/useTableNavigation/useTableNavigation';
+import { TableControls } from './components/TableControls/TableControls';
```

Comment:
Doesn't really make such a difference but we could have an index file exporting all the components inside components folder